python/ray/autoscaler/gcp/example-gpu-docker.yaml

```diff
@@ -25,16 +25,13 @@ autoscaling_mode: default
 docker:
     image: "rayproject/ray:0.8.7-gpu"
     container_name: "ray-nvidia-docker-test" # e.g. ray_docker
-    run_options:
-      - --runtime=nvidia
+    disable_automatic_runtime_detection: False
```

Comment:
Can you leave this out of the examples? It should be an internal flag that users don't need to use.